.github/workflows/main.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           dotnet-version: '6.0.x'
```

Comment:
I dunno, coincidentally with this branch being the first pass some unrelated ones started passing at the same time??

![image](https://user-images.githubusercontent.com/1588923/217398978-a1bd3aae-3596-4fea-8bfe-7dc56a9dcc3f.png)

Reply:
Tried a simple readme change in a branch and that passed.
Something sketchy goin on